### ISSUE TITLE
HackMD presentations with fade transition do not render

### ISSUE THREAD

**Issue by sedrubal** [astefanutti/decktape#84]
I tried to export a Reveal.js presentation from [hackmd](https://hackmd.io/), but I got a PDF with only black pages. I couldn't reproduce this with [hackmd example presentation](https://hackmd.io/p/slide-example#/) but with this presentation: https://hackmd.io/p/ByeVG31wl#/ (Markdown: https://hackmd.io/ByeVG31wl)

```sh
$> docker run -v `pwd`:/slides:rw,Z astefanutti/decktape https://hackmd.io/p/ByeVG31wl#/ slides.pdf
Loading page https://hackmd.io/p/ByeVG31wl#/ ...
Deprecation warning: value provided is not in a recognized ISO format. moment construction falls back to js Date(), which is not reliable across all browsers and versions. Non ISO date formats are discouraged and will be removed in an upcoming major release. Please refer to http://momentjs.com/guides/#/warnings/js-date/ for more info.
Arguments:
[0] _isAMomentObject: true, _isUTC: false, _useUTC: false, _l: undefined, _i: Wed Jan 25 2017 19:22:56 GMT+0000 (UTC), _f: undefined, _strict: undefined, _locale: [object Object]
undefined
Loading page finished with status: success
Reveal JS DeckTape plugin activated
Printing slide #/10     (22/22) ...
Printed 22 slides
```

The PDF contains 22 black pages.

**Comment by astefanutti** [comment]
The PhantomJS 'rasterize.js' example fails at exporting even the first slide:

    phantomjs examples/rasterize.js https://hackmd.io/p/ByeVG31wl#/ test.png

It seems there is something specific in your slides that prevents PhantomJS to render the page. As it works fine with other presentations, would you be able to isolate the minimal presentation that reproduces the issue?

**Comment by sedrubal** [comment]
The problem seems to be the

```yaml
---
slideOptions:
  transition: fade
```

at the top of the presentation. Removing `transition: fade` fixes the problem.

**Comment by astefanutti** [comment]
> But transition: fade is no problem for other Reveal.js presentations (both: a presentation from html file and a presentation hosted anywhere)

Do you mean it's specific to HackMD and that you can export other presentations with `transition: fade` to PDF successfully?

**Comment by sedrubal** [comment]
Yes.

**Comment by astefanutti** [comment]
I suspect this is due to the HackMD CSS that gets added over Reveal CSS:

```
.reveal.fade {
    opacity: unset;
    -webkit-transition: unset;
    transition: unset;
}
```
And maybe other HackMD customisations to Reveal that, for some reasons, PhantomJS is not capable of rendering.

I don't think there is much I'll be able to do about it. I plan to rebase DeckTape on the latest 2.5 version of PhantomJS which brings better CSS3 support, among many other things. So chances are this issue is going to be fixed in the process.

**Comment by astefanutti** [comment]
Should be fixed in version `2.0.1` that uses headless Chrome. Let me know if you still face this issue.
